pyproject.toml

```diff
@@ -24,6 +24,8 @@ packages = [
 python = "^3.9"
 torch = "^2.2.2"
 torchvision = "^0"
+transformers = "^4.46.2"
```

Comment:
RT-DETR was published in [4.42.0](https://github.com/huggingface/transformers/releases/tag/v4.42.0) would that be a good initial version?